code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -130,6 +131,68 @@
 		new /obj/item/reagent_containers/food/pill/initropidril(src)
 		new /obj/item/flashlight/flare/glowstick/red(src)
 
+/obj/item/storage/box/blueshield/laser
+	name = "Blueshield Equipment (Aegis SG7 stun revolver)"
+	desc = "It's a box containing an Aegis SG7 stun revolver."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "blu_box"
+
+	New()

```

Comment:
Stop this nonsense.

Reply:
I only did that because the sprite wasn't appearing when I didn't specify the icon, but I'll try it again without and see if I can figure out why the heck it's not working.